scania-digital-design-system/tegel
# feat(chip): prefix change

**Describe pull-request**  
Prefix change for Chip component

**Solving issue**  
Fixes: [DTS-1631](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?assignee=63ff6d6c314f5088138187f0&selectedIssue=DTS-1631)

**How to test**  
_Add description how to test if possible_
1. Go to Storybook
2. Check in Components -> Chip
3. Check the components APIs and styles.

**Suggested test steps**


[DTS-1631]: https://tegel.atlassian.net/browse/DTS-1631?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
